modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -125,7 +127,7 @@ const renderInlineDialog = <T extends Dialog.DialogData>(dialogInit: DialogManag
     components: [
       memHeader.asSpec(),
       memBody.asSpec(),
-      memFooter.asSpec()
+      ...optMemFooter.map((memFooter) => [ memFooter.asSpec() ]).getOr([])
```

Comment:
```suggestion
      ...optMemFooter.map((memFooter) => memFooter.asSpec()).toArray()
```
This will wrap the optional value and handle the none case.

Reply:
Good catch 👍